doc/TESTING.md

```diff
@@ -161,17 +161,28 @@ It is a staging server that includes a copy of the production database and is in
 Use it when you need database records that are too difficult or time-consuming to reproduce locally, or to debug a problem on the production site that you can't reproduce locally.
 See summary and usage instructions below.
 
-**Environment:** Test environment
-**Database:** MySQL
-**Data:** Data present at https://unstable.publiclab.org is updated occasionally using the Production database, so it has good amount of data for testing. 
-**Email testing:** _unstable_ server can't send emails so Email related features can't be tested.  
+**Environment:** Test environment<br>
+**Database:** MySQL<br>
+**Data:** Data present at https://unstable.publiclab.org is updated occasionally using the Production database, so it has good amount of data for testing. <br>
+**Email testing:** _unstable_ server can't send emails so Email related features can't be tested.  <br>
 
 ### Pushing on `unstable`
 
 1. Drop a message in the [Gitter chatroom](https://gitter.im/publiclab/publiclab) that you are pushing to _unstable_, as other members also use it for testing their work so you don't want to shock them by pushing your work over theirs. After dropping the message, wait for 5 mins, and if no one mentions that they are using the branch, then go to step 2.
-2. Push using: `git push -f https://github.com/publiclab/plots2.git HEAD:unstable`.<br><br>**Note:** If you get this message - `remote: Permission to publiclab/plots2.git denied to <username>` then you don't have permission to push to _unstable_ branch.
```

Comment:
You don't need to make any change here @anirudhprabhakaran3 .

Reply:
I've made the change. Thanks!